data/beaconrestapi/src/main/java/tech/pegasys/teku/beaconrestapi/handlers/tekuv1/beacon/GetEth1VotesStats.java

```diff
@@ -0,0 +1,213 @@
+              Eth1DataVotesStats::getVotesBreakdown)
+          .withField(
+              "win_votes_required", CoreTypes.UINT64_TYPE, Eth1DataVotesStats::getWinVotesRequired)
+          .withField(
+              "voting_period_slots",
+              CoreTypes.UINT64_TYPE,
+              Eth1DataVotesStats::getVotingPeriodSlots)
+          .withField(
+              "voting_period_slots_left",
+              CoreTypes.UINT64_TYPE,
+              Eth1DataVotesStats::getVotingPeriodSlotsLeft)
+          .build();
+
+  private static final SerializableTypeDefinition<Eth1DataVotesStats>
+      ETH1VOTES_STATS_RESPONSE_TYPE =
+          SerializableTypeDefinition.<Eth1DataVotesStats>object()
+              .name("GetEth1VotesStatsResponse")
+              .withField("data", ETH1VOTES_STATS_TYPE, Function.identity())
+              .build();
+
+  private final ChainDataProvider chainDataProvider;
+  private final Eth1DataProvider eth1DataProvider;
+
+  public GetEth1VotesStats(
+      final DataProvider dataProvider, final Eth1DataProvider eth1DataProvider) {
+    super(
+        EndpointMetadata.get(ROUTE)
+            .operationId("getEth1VotesStats")
+            .summary("Get Eth1DataVotes stats")
+            .description(
+                "Returns information about the current state of voting for Eth1Data from the specified state.")
+            .pathParam(PARAMETER_STATE_ID)
+            .tags(TAG_TEKU)
+            .response(SC_OK, "Request successful", ETH1VOTES_STATS_RESPONSE_TYPE)
+            .response(SC_NOT_FOUND, NOT_FOUND_MESSAGE)
+            .build());
+    this.chainDataProvider = dataProvider.getChainDataProvider();
+    this.eth1DataProvider = eth1DataProvider;
+  }
+
+  @OpenApi(
+      path = OAPI_ROUTE,
+      method = HttpMethod.GET,
+      summary = "Get Eth1DataVotes stats",
+      description =
+          "Returns information about the current state of voting for Eth1Data from the specified state.",
+      pathParams = {@OpenApiParam(name = PARAM_STATE_ID, description = PARAM_STATE_ID_DESCRIPTION)},
+      tags = {TAG_TEKU},
+      responses = {
+        @OpenApiResponse(
+            status = RES_OK,
+            content = @OpenApiContent(from = GetEth1VotesStatsResponse.class)),
+        @OpenApiResponse(status = RES_NOT_FOUND, description = NOT_FOUND_MESSAGE),
+        @OpenApiResponse(status = RES_INTERNAL_ERROR)
+      })
+  @Override
+  public void handle(@NotNull final Context ctx) throws Exception {
+    adapt(ctx);
+  }
+
+  @Override
+  public void handleRequest(final RestApiRequest request) throws JsonProcessingException {
+    request.respondAsync(
+        chainDataProvider
+            .getBeaconStateAndMetadata(request.getPathParameter(PARAMETER_STATE_ID))
+            .thenApply(
+                maybeStateAndMetadata -> {
+                  if (maybeStateAndMetadata.isEmpty()) {
+                    return AsyncApiResponse.respondWithError(SC_NOT_FOUND, NOT_FOUND_MESSAGE);
```

Comment:
This isn't the standard format for error responses which should be:

```
{
  "code": 404,
  "message": "Not found"
}
```

Can just use the helper function:
```suggestion
                    return AsyncApiResponse.respondNotFound();
```